iina/PlayerWindowController.swift

```diff
@@ -255,11 +258,51 @@ class PlayerWindowController: NSWindowController, NSWindowDelegate {
 
   override func keyDown(with event: NSEvent) {
     let keyCode = KeyCodeHelper.mpvKeyCode(from: event)
-    if let kb = PlayerCore.keyBindings[keyCode] {
+    if keyCode != "" {
+      #if DEBUG
+      Logger.log("KeyDown: \(keyCode)", level: .verbose)
+      #endif
+      if matchAndHandle(keyCode) {
+        return
+      }
+
+      // try to match key sequences, up to 4 values. shortest match wins
```

Comment:
A developer who is unaware of mpv's support of key sequences may be very confused by this code as we are not adding full support for this feature. If you go to Preferences / Key Bindings and try and add a key sequence you can't, yes? A developer thinking about the IINA key binding UI may wonder what this is about. Might want to indicate this is for users that edit their input.conf file by hand to take advantage of mpv features not yet supported by the IINA key binding UI.